.circleci/config.yml

```
@@ -10,7 +10,11 @@ jobs:
       - run:
           name: "Build COMPASS docker image"
           command: |
-            docker build . -f docker/Dockerfile
+            docker build . -f docker/Dockerfile -t test_image
+      - run:
+          name: "Run integration tests"
+          command: |
+            docker run test_image bash -c "cd $HOME/OPERA/COMPASS && python -m pytest"
```

```suggestion
            docker run test_image bash -c "cd ~/OPERA/COMPASS && python -m pytest"
```

Maybe this will work? Looks like $HOME is being evaluated in the docker host and giving the wrong directory